staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresourcedefinition/etcd.go

```diff
@@ -92,6 +92,16 @@ func (r *REST) Delete(ctx context.Context, name string, options *metav1.DeleteOp
 		)
 		return nil, false, err
 	}
+	if options.Preconditions.ResourceVersion == nil {
+		options.Preconditions.ResourceVersion = &crd.ResourceVersion
```

Comment:
I don't think pinning an unspecified resource version to the pre-fetched instances resource version is correct here... I would leave this nil if unspecified

Reply:
Oh, this makes sense. I thought RV check should be always mandatory.